datatypes.go

```diff
@@ -28,6 +29,14 @@ const (
 const (
 	format        = "15:04 02.01.2006"
 	remindMessage = "This metric has been in bad state for more than %v hours - please, fix."
+	limit         = 1000
+)
+
+type notificationEventSettings int
+
+const (
+	DefaultNotificationSettings notificationEventSettings = iota
+	ShortMessage
```

Comment:
Maybe it would be more clear if we name it like that?

```suggestion
	SIFormatNumbers
```

Reply:
Ok, I've fixed it